tests/callbacks/test_early_stopping.py

```diff
@@ -359,8 +359,13 @@ def validation_epoch_end(self, outputs):
     by_min_epochs = min_epochs * limit_train_batches
 
     # Make sure the trainer stops for the max of all minimum requirements
-    assert trainer.global_step == max(min_steps, by_early_stopping, by_min_epochs), \
-        (trainer.global_step, max(min_steps, by_early_stopping, by_min_epochs), step_freeze, min_steps, min_epochs)
+    assert trainer.global_step == max(min_steps, by_early_stopping, by_min_epochs), (
+        trainer.global_step,
+        max(min_steps, by_early_stopping, by_min_epochs),
+        step_freeze,
+        min_steps,
+        min_epochs,
```

Comment:
can we have some better error formatting at least say what are the variables, this is just bunch of numbers...

Reply:
@Borda I think this is just the auto formatter and there is actually no change in this PR.

Reply:
too late to bring up using `black` as the formatter? 😅

Reply:
@Borda 😈

Reply:
@ananthsub you can use @carmocca way... format `black` (conservative) and the `yapf` (liberal) :P